style.css

```diff
@@ -0,0 +1,100 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+@media (max-width: 768px) {
+  main {
+    background-image: url(assets/header-illsutration-mobile.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  animation-duration: 0.5s;
+  animation-name: obj-scale;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  animation-duration: 0.5s;
+  animation-name: obj-scale;
+}
```

Comment:
- [ ] Please 🙏🏽 use the correct background image to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://github.com/Kingfash084412/Portfolio/assets/58642949/03b34c18-4305-4515-8a8c-1c2438caa7f8) | ![image](https://github.com/Kingfash084412/Portfolio/assets/58642949/54943557-6dbe-4459-b56b-17cd4814c086) |